### DISCUSSION TITLE
Audible activation code.

### DISCUSSION THREAD

**Discussion by PaulJHannah** [audiamus/AaxAudioConverter#98] [Q&A] [answered]
Hi, 
I have already got the app installed but it didn't find the code, what do I do now?

**Comment by audiamus** [comment]
Download the official Audible App for Windows 10, available from the Windows Store, and log-in to your Audible account via that app once. This will download your activation code. You can then close the Audible app. Subsequently, AAX Audio Converter should be able to find that activation code. 
Nonetheless, I would suggest to also download your books with the Audible app, not with the web browser, as app downloads come with additional meta data, like explicit chapter titles or series info.

_I have AAX Audio Converter 2 in the making which will offer an alternative option: To directly access your Audible account from the converter and to download with the converter itself. It will decrypt the audio on-the-fly during download. That method will also include the extra meta data._

**Comment by PaulJHannah** [comment]
So... are you saying I must run your program and do my conversions on my
phone?
Paul Hannah

*The preceding text of this email may be confidential and or privileged. If
it is, keep it to yourself, we wouldn't want this stuff getting around,
would we? *

**Reply by audiamus** [reply]
As I said, the official Audible App for Windows 10. Nothing to do with Windows Phone.

**Comment by PaulJHannah** [comment]
Okay,  I have the official audible app on my win 10 machine. On the same
machine I subsequently downloaded your app, and it didn't find it.
Thanks.
Paul

**Reply by audiamus** [reply]
With the Audible app, could you successfully log-in to your Audible account? The Audible app should list all your books then.

**Comment by PaulJHannah** [comment]
Yes, I  use it  often .

**Comment by PaulJHannah** [comment, marked as answer]
Thanks for your help, I rebooted and it worked.

Paul Hannah

*The preceding text of this email may be confidential and or privileged. If
it is, keep it to yourself, we wouldn't want this stuff getting around,
would we? *
